Bugzilla/CGI.pm

```diff
@@ -441,12 +441,20 @@ sub header {
         %headers = ('-type' => shift(@_));
     }
     else {
-        %headers = @_;
+        # we always want headers to be in lower case, to avoid
+        # for instance -Cache_Control vs. -cache_control.
+        my %tmp = @_;
+        foreach my $key (keys %tmp) {
+            my $lc_key = lc $key;
+            warn "duplicate header: $key" if exists $headers{$lc_key};
```

Comment:
Advise warn "overwriting duplicate header: $key" to clearly state the actual warning.